webpack/webpack.dev.config.js

```diff
@@ -0,0 +1,29 @@
+var webpack = require('webpack');
+var path = require('path');
+
+var parentDir = path.join(__dirname, '../');
+
+module.exports = {
+	entry: [
```

Comment:
A nên setting 1 tab = 2 spaces trong vscode nhé https://stackoverflow.com/questions/29972396/how-to-set-tab-space-style

Reply:
Done!